_data/projects/Coverlet.yml

```diff
@@ -0,0 +1,16 @@
+name: Coverlet
+desc: Cross platform code coverage for .NET
+site: https://github.com/tonerdo/coverlet
+tags:
+- cross-coverage
+- dotnet-core
+- dotnet-cli
+- unittest
+- coverage
+- cross-platform
+- coverage-library
+- dotnet
+- coverlet
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/tonerdo/coverlet/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```

Comment:
@MarcoRossignoli Please change the link to https://github.com/tonerdo/coverlet/labels/up-for-grabs

Reply:
done